src/ui-components/input-field/input-field.spec.js

```diff
@@ -0,0 +1,315 @@
+import React from "react";
+import { shallow } from "enzyme";
+import InputField from "./input-field.js";
+import { CSS_CLASSES } from "./constants.js";
+
+describe("Input field", () => {
+  
+  it("mounts component", () => {
+    const wrapper = shallow(<InputField />);
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("is not focused", () => {
+    const wrapper = shallow(<InputField />);
+    expect(wrapper.state().isInputFieldFocused).toEqual(false);
+  })
+
+  describe("props are set correctly", () => {
+
+    function testInputFieldProps(givenData, expectedResults) {
+      const wrapper = shallow(
+        <InputField
+          id={givenData.id}
+          className={CSS_CLASSES.FIELD}
+          type={givenData.type}
+          required={givenData.required}
+          value={givenData.value}
+          />);
+  
+      expect(wrapper.find("input").props().id).toEqual(expectedResults.id);

```

Comment:
Here we are calling `wrapper.find("input").props()` 5 times. We could extract that call to a variable like `const inputProps = wrapper.find("input").props()` and then proceed with expects.

- [ ] Extract find call to a local variable.